README.md

```diff
@@ -2,7 +2,28 @@
 
 This repository contains the various components and some documentation for the Plutus Platform, providing smart contract functionality for the Cardano blockchain.
 
+## Building
+
+`default.nix` defines a package set containing all the packages in this repository. These can be built directly. 
+For example:
+```
+nix build -f default.nix language-plutus-core
+```
+
+## Developing
+
+An appropriate environment for developing a package can be entered using the `env` attribute of the package. For example:
+```
+nix-shell default.nix -A language-plutus-core.env
+```
+
+## Updating the generated Haskell package set
+
+`pkgs/default.nix` contains a generated package set with all the dependencies for this projet.
+
+You should regenerate this if you change any dependencies in cabal files. To do this, use the 
```

Comment:
Looks like there is a word missing after "the".

Reply:
Fixed, thanks.